pages/design/rule-template.md

```diff
@@ -117,7 +117,7 @@ Description...
 
 Composite rules are rules that take results from different rules and through some logic come to a single result. For example: SC 1.2.3 allows video to pass with either a transcript, an audio description, or if it is a media alternative. Each of these would be atomic rules, that are used in a composite rule. The composite rule than describes that at least one of these must pass for the composite rule to pass.
 
-For more about composite rules, see the [ACT Rules Format](https://www.w3.org/TR/act-rules-format/#composed-rules). To create a composite rule, use the [empty composite rule template](./composite-template-empty.html).
+For more about composite rules, see the [ACT Rules Format](https://www.w3.org/TR/act-rules-format/#composed-rules). To create a composite rule, use the [empty composite rule template](https://raw.githubusercontent.com/act-rules/act-rules.github.io/develop/pages/design/composite-template-empty.md).
```

Comment:
Same remark: shouldn't we use `/pages/design/composite-template-empty/`?

Reply:
See above comment, additionally I have updated the GitHub templates to refer to the same (one source of truth).

See - https://github.com/act-rules/act-rules.github.io/pull/648/files#diff-1a1c3dd142f76a5fad803a0c52839881R5